### PR TITLE
Feature/blog post page

### PR DESCRIPTION
**Pull request by yosser**

### CODE REVIEW

**Comment by andgreendigital** [comment]
Can you update this PR? 🙂

File: src/templates/blog-post-template.jsx [hunk 1]
```diff
@@ -183,35 +194,77 @@ query BlogPostBySlug($slug: String!) {
...
+};
+
+const BlogPostDescriptionSection = ({
+    author, publishDate, timeToRead, description,
+}) => (
+    <div className={css.blogPost_title_block}>
+        <div className={css.blogPost_title_left}>
+            <p className={css.blogPost_author}>by {author}</p>
+            <p className={css.blogPost_timings}>{publishDate}</p>
+            <p className={css.blogPost_timings}>{timeToRead} min read [TBC]</p>
```
Comment (andgreendigital) [inline src/templates/blog-post-template.jsx]: Why is there a TBC here? 🙂

File: src/templates/blog-post-template.jsx [hunk 2]
```diff
@@ -183,35 +194,77 @@ query BlogPostBySlug($slug: String!) {
...
+                timeToRead={post.body.childMarkdownRemark.timeToRead}
+                description={post.description.description}
+            />
+            <p style={{ display: 'block' }} />
             {
                 components && components.map(renderComponent)
             }
-            <Footer />
+            <div className={css.blogPost_tag_container}>
+          [TBC] {tags.map((tag) => <span key={tag} className={css.blogPost_tag}>{tag}</span>)}
```
Comment (andgreendigital) [inline src/templates/blog-post-template.jsx]: Why is this TBC here? 🙂

Comment (andgreendigital) [reply src/templates/blog-post-template.jsx]: Also I don't see tags on the wireframes so need to follow up how these should look! 🙂

Comment (yosser) [reply src/templates/blog-post-template.jsx]: See the story

Comment (andgreendigital) [reply src/templates/blog-post-template.jsx]: Could you send a link for the wireframe you are referring to?

File: src/templates/blog-post-template.jsx [hunk 3]
```diff
@@ -205,19 +257,21 @@ const BlogPostTemplate = ({ data }) => {
...
-                {post.publishDate}
-            </p>
+            <BlogPostTitleSection breadcrumbs="Put breadcrumbs here" title={post.title} />
+            <BlogPostDescriptionSection
+                author={post.author.name}
+                publishDate={post.publishDate}
+                timeToRead={post.body.childMarkdownRemark.timeToRead}
+                description={post.description.description}
+            />
+            <p style={{ display: 'block' }} />
```
Comment (andgreendigital) [inline src/templates/blog-post-template.jsx]: Can we keep styling in css files?

File: src/templates/blog-post-template.module.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
+.hero {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 12.5em;
+  background: #e1e1e1;
+  margin: -1em -2.5em 1em;
```
Comment (anasufana) [inline src/templates/blog-post-template.module.css]: The blog hero is just an image container so it doesn't need all these properties.

File: src/templates/blog-post-template.module.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+.blogPost_tag {
+  border: 1px dotted lightblue;
+  padding: 7px 4px;
+  margin: 10px 10px;
+}
+
+.blogPost_breadcrumbs {
+  font-size: 22px;
+  line-height: normal;
```
Comment (anasufana) [inline src/templates/blog-post-template.module.css]: We've added all the headings & body text sizes into our base.css.

Can you use the appropriate class on the breadcrumb element to apply it?

File: src/templates/blog-post-template.module.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.blogPost_author{
+  font-family: Poppins;
+  font-size: 32px;
+  font-weight: bold;
+  line-height: normal;
+  letter-spacing: 0.53px;
+  color: var(--off-black);
+  margin-bottom: 10px;
```
Comment (anasufana) [inline src/templates/blog-post-template.module.css]: same as above

File: src/templates/blog-post-template.module.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.blogPost_timings{
+  font-size: 22px;
```
Comment (anasufana) [inline src/templates/blog-post-template.module.css]: same as above

File: src/templates/blog-post-template.module.css [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+  line-height: normal;
+  letter-spacing: 0.37px;
+  color: var(--off-black);
+  margin-bottom: 10px;
+}
+
+.blogPost_body {
+  font-size: 22px;
```
Comment (anasufana) [inline src/templates/blog-post-template.module.css]: same as above

File: src/templates/blog-post-template.jsx [hunk 9]
```diff
@@ -189,9 +200,47 @@ query BlogPostBySlug($slug: String!) {
...
+    title: PropTypes.string,
+};
 
+const BlogPostDescriptionSection = ({
+    author, publishDate, timeToRead, description,
+}) => (
+    <div className={css.blogPost_description_block}>
+        <div className={css.blogPost_title_left}>
+            <h3 className={css.blogPost_author}>by {author}</h3>
+            <p className={css.blogPost_timings}><span className="font_style__B1">{publishDate}</span></p>
```
Comment (anasufana) [inline src/templates/blog-post-template.jsx]: is wrapping the text in a span necessary? the `className` can be added like this 

```className={`${css.blogPost_timings} font_style__B1`}```

Comment (yosser) [reply src/templates/blog-post-template.jsx]: It's because of your margin

**Pull request merged**